airflow/www/views.py

```diff
@@ -2298,13 +2299,24 @@ def _mark_dagrun_state_as_queued(self, dag_id: str, dag_run_id: str, confirmed:
         if not dag:
             return {"status": "error", "message": f"Cannot find DAG: {dag_id}"}
 
-        new_dag_state = set_dag_run_state_to_queued(dag=dag, run_id=dag_run_id, commit=confirmed)
+        set_dag_run_state_to_queued(dag=dag, run_id=dag_run_id, commit=confirmed)
 
         if confirmed:
             return {"status": "success", "message": "Marked the DagRun as queued."}
 
         else:
-            details = [str(t) for t in new_dag_state]
+            # Identify tasks that will be queued up to run when confirmed
+            all_task_ids = [task.task_id for task in dag.tasks]
+
+            existing_tis = session.query(TaskInstance.task_id).filter(
+                TaskInstance.dag_id == dag.dag_id,
+                TaskInstance.run_id == dag_run_id,
+                TaskInstance.state is not None,
```

Comment:
Do we need to check for `state is not None`? Aren't we looking for tasks that don't have a corresponding instance for this run?

Reply:
@bbovenzi I've removed this check. When testing. I had thought I'd seen it get into a state where a task would end up in the database but the state was None. I can't reproduce this so have removed it.